packages/amplify-category-geo/src/provider-controllers/import.ts

```diff
@@ -0,0 +1,143 @@
+    if (!collection.output || !collection.output.Name || !collection.output.Region) {
+      return false;
+    }
+    return true;
+  });
+  if (provisionedCollectionNames.length === 0) {
+    throw new Error('No geofence is not provisioned yet. Run \`amplify push\` to provision geofence collection.')
+  }
+  // Get collection region
+  const collectionRegion = geofenceCollectionResourcesMap[provisionedCollectionNames[0]].output.Region;
+  // Get collection output name
+  const provisionedCollectionOutputNames = provisionedCollectionNames.map(collectionName => geofenceCollectionResourcesMap[collectionName].output.Name);
+  // Get the collection to import
+  let collectionToImport: string = provisionedCollectionOutputNames[0];
+  if (provisionedCollectionOutputNames.length > 1) {
+    if (provisionedCollectionOutputNames.length < collectionNames.length) {
+      printer.warn('There are additional geofence collections in the project that have not been deployed. To import data into these resources, run `amplify push` first.');
+    }
+    collectionToImport = await prompter.pick<'one', string>('Select the Geofence Collection to import with Geofences', provisionedCollectionOutputNames);
+  }
+  // Ask for geo json file path
+  let geoJSONFilePath: string;
+  geoJSONFilePath = join(await prompter.input(`Provide the path to GeoJSON file containing the Geofences for ${collectionToImport} collection. Refer https://geojson.io/ for a sample GeoJSON:`));
+  while (!existsSync(geoJSONFilePath)) {
+    geoJSONFilePath = join(await prompter.input(`Cannot find GeoJSON file. Re-enter a valid file path:`));
+  }
+  let geoJSONObj: FeatureCollection;
+  geoJSONObj = JSON.parse(readFileSync(geoJSONFilePath, 'utf-8')) as FeatureCollection;
+  const identifierWalkthroughOptions = [
+    { name: 'Use root-level "id" field. (Auto-assigned if missing. This will MODIFY the GeoJSON file)', value: { identifierType: IdentifierOption.RootLevelID, identifierField: 'id' } },
+    ...scanCandidateCustomProperties(geoJSONObj).map(prop => ({
+      name: prop,
+      value: {
+        identifierType: IdentifierOption.CustomProperty,
+        identifierField: prop,
+      },
+    })),
+  ];
+  const { identifierField, identifierType } = await prompter.pick<'one', any>('Select the property to use as the Geofence feature identifier:', identifierWalkthroughOptions) as IdentifierInfo;
+  // Validate the json file against schema
+  printer.info('Validating your GeoJSON file...');
+  try {
+    geoJSONObj = validateGeoJSONObj(geoJSONObj, identifierField, identifierType);
+    printer.success('Successfully validated GeoJSON file.');
+  } catch (err) {
+    printer.error('Error occurs while validating GeoJSON file.');
+    throw err;
+  }
+  // Update the GeoJSON file with auto-assigned ID
+  if (identifierType === IdentifierOption.RootLevelID) {
+    writeFileSync(geoJSONFilePath, JSON.stringify(geoJSONObj, null, 2));
```

Comment:
use `JSONUtilities`:
```suggestion
    JSONUtilities.writeJson(geoJSONFilePath, geoJSONObj);
```